src/react-apps/applications/receipt/src/utils/urlHelper.ts

```diff
@@ -0,0 +1,49 @@
+import { getInstanceId, getInstanceOwnerId } from './instance';
+
+export const altinnAt21PlatformUrl = 'https://platform.at21.altinn.cloud/';
+export const altinnAt21Url = 'https://at21.altinn.cloud/';
+export const altinnUrl = 'http://altinn.no/';
+export const altinnOrganisationsUrl = 'https://altinncdn.no/orgs/altinn-orgs.json';
+export const languageUrl = `${window.location.origin}/receipt/api/v1/language/GetLanguageAsJSON?languageCode=nb`;
+
+export function getInstanceMetadataUrl(): string {
+  return `${getAltinnCloudUrl()}storage/api/v1/instances/${getInstanceOwnerId()}/${getInstanceId()}`;
+}
+
+export function getPartyUrl(): string {
+  return `${getAltinnCloudUrl()}register/api/v1/parties/${getInstanceOwnerId()}`;
+}
+
+export function getAltinnCloudUrl(): string {
+  if (window.location.hostname === 'localhost'
+    || window.location.hostname === '127.0.0.1'
+    || window.location.hostname === 'altinn3.no'
+  ) {
+    // if we are developing locally, point to test data in at21
+    return altinnAt21PlatformUrl;
+  } else {
+    // in prod => point to origin. Can be multiple environments.
```

Comment:
*adjusts glasses*
Uhm, actually, this will also say that it is in development.

Reply:
:(